drivers/input/keyboard/miyoo_kbd.c

```diff
@@ -1113,7 +1124,10 @@ static void __exit kbd_exit(void)
 {
   input_unregister_device(mydev);
   del_timer(&mytimer);
-
+  if (miyoo_increase_volume)
+	  symbol_put(MIYOO_INCREASE_VOLUME);
+  if (miyoo_decrease_volume)
+	  symbol_put(MIYOO_DECREASE_VOLUME);
```

Comment:
since this is a keyboard module and to not mix stuff we could remove BACKLIGHT and VOLUME modification here and leave it for deamon (majority will probably use that in the end). What's your opinion?

Reply:
Agree